packages/react/src/components/DatePickerInput/DatePickerInput.js

```diff
@@ -218,9 +212,8 @@ export default class DatePickerInput extends Component {
       return (
         <Calendar16
           className={`${prefix}--date-picker__icon`}
-          aria-label={iconDescription}
-          onClick={openCalendar}
-          role="img">
+          role="img"
+          aria-hidden="true">
```

Comment:
Do we still need to still call this function `onClick`, so it's not a breaking change in functionality? 

It's no longer being used in `DatePicker` to actually open the calendar but maybe it's possible someone is relying on this function being called when the calendar is clicked vs the input?

Reply:
makes sense to leave it on while the prop is being deprecated 👍🏾

Reply:
I think we can still remove DatePicker from calling it though. The `openCalendar` function is never called when the icon is clicked since it now has pointer-events none

https://github.com/carbon-design-system/carbon/blob/e8c23254039993e2fcaae9b558f3ae55df7839b9/packages/react/src/components/DatePicker/DatePicker.js#L532-L536